Fix the link provided in the GitButler workspace commit.

Byron opened the pull request:
Maybe we could also have a redirect from

https://docs.gitbutler.com/features/virtual-branches/integration-branch

to

https://docs.gitbutler.com/features/branch-management/integration-branch

Byron committed:
Fix the link provided in the GitButler workspace commit.

Byron commented:
@estib-vega @PavelLaptev Would it be good to have a redirect from the old to the new link?
If it's a hassle, it's probably not worth it though as all these workspace commits get rewritten all the time, so the fix in this PR should do the trick.
Maybe a redirect would be a good safety measure in case the old link is used in more places.

PavelLaptev commented:
@Byron good catch! Let's merge this one. And I'll add a redirect thingy to the docs

Byron committed:
Try to fix playwright by removing extra Rust cache

Otherwise, all dependencies seem to be missing:

```
   Compiling gitbutler-oxidize v0.0.0 (/home/runner/work/gitbutler/gitbutler/crates/gitbutler-oxidize)
   Compiling gitbutler-fs v0.0.0 (/home/runner/work/gitbutler/gitbutler/crates/gitbutler-fs)
   Compiling gitbutler-id v0.0.0 (/home/runner/work/gitbutler/gitbutler/crates/gitbutler-id)
   Compiling gitbutler-tagged-string v0.0.0 (/home/runner/work/gitbutler/gitbutler/crates/gitbutler-tagged-string)
error[E0463]: can't find crate for `gix`
 --> crates/gitbutler-oxidize/src/lib.rs:4:5
  |
4 | use gix::bstr::ByteSlice;
  |     ^^^ can't find crate

error[E0463]: can't find crate for `gix`
 --> crates/gitbutler-fs/src/lib.rs:9:5
```
